scenarios/base-2050.yml

```diff
@@ -0,0 +1,84 @@
+# This scenario config file contains information for several steps in the model pipeline, i.e.
+# build_datapackage, parametrize, optimize and potentially also postprocess or plot.
+
+# build_datapackage
+
+name: base-2050
+
+regions:
+  - BB
+  - B
+
+links:
+  - BB-B
+
```

Comment:
Seems like this file is missing 'datetimeindex'. 

```
Traceback (most recent call last):
  File "scripts/build_datapackage.py", line 238, in <module>
    start=scenario_specs["datetimeindex"]["start"],
KeyError: 'datetimeindex'
```

scenario_specs is the loaded data of `scenarios/base-2050.yml`